content/applications/hr/appraisals/reporting.rst

```diff
@@ -0,0 +1,116 @@
+
+.. _appraisals/skills-report:
+
+Skills evolution
+----------------
+
+To access the *Skills Evolution* report, navigate to :menuselection:`Appraisals application -->
+Reporting --> Skills Evolution`. This displays a report of all skills, grouped by employee.
+
+Skill levels are **only** updated after an appraisal is marked as done. Any skill level changes from
+any ongoing appraisals that have **not** been finalized are **not** included in this report.
+
+All the lines of the report are collapsed, by default. To view the details of a line, click anywhere
+on a line to expand the data.
+
+Each skill has the following information listed:
+
+- :guilabel:`Employee`: name of the employee.
+- :guilabel:`Skill Type`: the category the skill falls under.
+- :guilabel:`Skill`: the specific, individual skill.
+- :guilabel:`Previous Skill Level`: the level the employee had previously achieved for the skill.
+- :guilabel:`Previous Skill Progress`: the previous percentage of competency achieved for the skill
+  (based on the :guilabel:`Skill Level`).
+- :guilabel:`Current Skill Level`: the current level the employee has achieved for the skill.
+- :guilabel:`Current Skill Progress`: the current percentage of competency achieved for the skill.
+- :guilabel:`Justification`: any notes entered on the skill explaining the progress.
+
+The color of the skill text indicates any changes from the previous appraisal. Skill levels that
+have *increased* since the last appraisal appear in green, skill levels that have *not changed*
+appear in black, and skills that have *regressed* appear in red.
+
+.. image:: reporting/skills-report.png
+   :align: center
+   :alt: A report showing all the skills grouped by employee.
+
+Filters
+~~~~~~~
+
+It is possible to only view skills that have changed over time, by using one of the preconfigured
+filters for the :guilabel:`Appraisal Skills Report`.
+
+To view the available filters, click the  :icon:`fa-caret-down` :guilabel:`(caret down)` icon in the
+:guilabel:`Search...` box.
+
+.. image:: reporting/filter.png
+   :align: center
+   :alt: A report showing all the skills grouped by employee.
+
+In the :guilabel:`Filters` section on the left-side of the drop-down menu, three options are
+presented:
+
+- :guilabel:`Improvement`: select this filter to **only** present skills that have increased since
+  the last appraisal. All skills appearing with this filter have green text, indicating the skill
+  level has improved.
+- :guilabel:`No Change`: select this filter to **only** present skills that have remained at the
+  same skill level since the last appraisal. All skills appearing with this filter have black text,
+  indicating there has been no change in the skill level.
+- :guilabel:`Regression`: select this filter to **only** present skills that have decreased since
+  the last appraisal. All skills appearing with this filter have red text, indicating the skill
+  level has dropped.
```

Comment:
Do you think it would be beneficial to mention that multiple options can be selected here?